README.md

```diff
@@ -4,19 +4,19 @@
 
 > A simple App that displays scores submitted by different players. It also allows you to submit your score.
 
-![screenshot](./)
+![screenshot](./screenshot.JPG)
 
 ## Built With
 
 - Javascript, HTML and CSS
 
 ## Live Demo
 
-[Live Demo Link]()
+[Live Demo Link](https://medaminedev66.github.io/leaderboard/)
 
 ## Getting Started
 
-- In terminal or Command line. Navigate to a directory in your local storage where you'd like to store the project and paste ```https://github.com/medaminedev66/to-do-list.git```
+- In terminal or Command line. Navigate to a directory in your local storage where you'd like to store the project and paste ```https://github.com/medaminedev66/leaderboard.git```
```

Comment:
- [ ] Kindly update the readme with ```git clone https://github.com/medaminedev66/leaderboard.git``` because just pasting the repo link in the terminal will not clone the project, also if project is on a branch `git checkout 'branchname'` should also be added as well. 😄

Reply:
Hi @ashakae, Thank you for reviewing my project. This Readme file is not for this branch `"final-touches"`, it's for the default branch (`develop`), the branch that I'm going to merge into, which means I should not add `git checkout 'final-touches'`.
I will update readme with `git clone https://github.com/medaminedev66/leaderboard.git`.